packages/web-runtime/src/index.ts

```diff
@@ -21,18 +21,22 @@ import {
 
 export const bootstrap = async (configurationPath: string): Promise<void> => {
   const runtimeConfiguration = await requestConfiguration(configurationPath)
-  announceOwncloudSDK({ vue: Vue, runtimeConfiguration })
-  await announceClient(runtimeConfiguration)
-  await announceStore({ vue: Vue, store, runtimeConfiguration })
-  await announceApplications({
+  const promiseOcSDK = announceOwncloudSDK({ vue: Vue, runtimeConfiguration }) // vue.$client
```

Comment:
can you explain why the awaits not in place anymore and instead happen right after? A possible optimization could be to parallelize the theme and translation loading `Promise.all([])`

Reply:
sorry i do not get the point, you await the returned function promise later which is the same es doing it right after calling the function. Can you please explain what you to do?

Reply:
Please see the previous conversation. Parallelising calls is not the same thing as running them sequentially.